packages/shadow.rb

```diff
@@ -3,30 +3,40 @@
 class Shadow < Package
   description 'Shadow password file utilities'
   homepage 'https://github.com/shadow-maint/shadow'
-  version '4.8.1'
+  @_ver = '4.8.1'
+  version "#{@_ver}-1"
   compatibility 'all'
-  source_url 'https://github.com/shadow-maint/shadow/releases/download/4.8.1/shadow-4.8.1.tar.xz'
+  source_url "https://github.com/shadow-maint/shadow/releases/download/#{@_ver}/shadow-#{@_ver}.tar.xz"
   source_sha256 'a3ad4630bdc41372f02a647278a8c3514844295d36eefe68ece6c3a641c1ae62'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/shadow-4.8.1-1-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '52d073915c219d4402907e47a32e89c1f61585c36d4cb608fcb6bcb03ed4f44e',
-     armv7l: '52d073915c219d4402907e47a32e89c1f61585c36d4cb608fcb6bcb03ed4f44e',
-       i686: 'e7559741bb901a45e9d98fd989750d6df0501611e81fa56af83014016df7e29a',
-     x86_64: '562849baf46b40c5e015c369ae77cc2f92266309acbc2b0cc1e87c0f0fa7a72b',
+  binary_sha256({
+    aarch64: '2f7ed952af80df7d0d57c68b1983d8de34fa29a8f66eec28ecd41f9efafea1c0',
+     armv7l: '2f7ed952af80df7d0d57c68b1983d8de34fa29a8f66eec28ecd41f9efafea1c0',
+       i686: 'a4d2a90a7993693ccb2935dadd9c4af1bc1465fe20868df7b1255a3603cfb9a2',
+     x86_64: 'd267579cb8a22d4a6378cd7c69033197cb685a7f2fc054b95707db3e400f8b8f'
   })
 
-  depends_on 'openpam'
+  depends_on 'linux_pam'
```

Comment:
Should we get rid of openpam?  I know these packages have conflicting files.

Reply:
Yes.